### PR TITLE
Add UITabBarController, add 5 ViewControllers, add labels in 4 vc

### PR DESCRIPTION
**Pull request by Nam-Namazov**

### CODE REVIEW

**Comment by Nam-Namazov** [comment]
Hey, Boss, check my pull request pls @N-ihad

File: homeWorkApp/homeWorkApp/Application/SceneDelegate.swift [hunk 1]
```diff
@@ -19,8 +19,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         guard let windowScene = (scene as? UIWindowScene) else { return }
         
         window = UIWindow(windowScene: windowScene)
-        window?.rootViewController = ViewController()
+        window?.rootViewController = MainTabBarController()
         window?.makeKeyAndVisible()
+        window?.overrideUserInterfaceStyle = .light
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Application/SceneDelegate.swift:24]: what's that for

File: homeWorkApp/homeWorkApp/Controllers/AboutViewController.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+//  AboutViewController.swift
+//  homeWorkApp
+//
+//  Created by Намик on 5/13/22.
+//
+
+
+import UIKit
+
+class AboutViewController: UIViewController {
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/AboutViewController.swift:11]: make class **final** if you don't intend others to inherit from your class

File: homeWorkApp/homeWorkApp/Controllers/AboutViewController.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    
+    private var label4 = UILabel()
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/AboutViewController.swift:13]: give a meaningful name and make it **let**

File: homeWorkApp/homeWorkApp/Controllers/AboutViewController.swift [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        super.viewDidLoad()
+        
+        style()
+        layout()
+    }
+    
+    private func style() {
+        view.backgroundColor = .white
+        title = "About"
+        label4.textColor = .black
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/AboutViewController.swift:25]: move this to where label4 is defined and initialize and change properties in a closure

File: homeWorkApp/homeWorkApp/Controllers/ContactsViewController.swift [hunk 5]
```diff
@@ -0,0 +1,38 @@
+//
+//  ContactsViewController.swift
+//  homeWorkApp
+//
+//  Created by Намик on 5/13/22.
+//
+
+import UIKit
+
+class ContactsViewController: UIViewController {
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/ContactsViewController.swift:10]: same here as before, make **final** if you don't intend others to inherit from your class

File: homeWorkApp/homeWorkApp/Controllers/ContactsViewController.swift [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    
+    private var label2 = UILabel()
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/ContactsViewController.swift:12]: same as before, change to **let**

File: homeWorkApp/homeWorkApp/Controllers/ContactsViewController.swift [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        super.viewDidLoad()
+        
+        style()
+        layout()
+    }
+    
+    private func style() {
+        view.backgroundColor = .white
+        title = "Contacts"
+        label2.textColor = .black
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/ContactsViewController.swift:24]: same

File: homeWorkApp/homeWorkApp/Controllers/HelpViewController.swift [hunk 8]
```diff
@@ -0,0 +1,38 @@
+//
+//  HelpViewController.swift
+//  homeWorkApp
+//
+//  Created by Намик on 5/13/22.
+//
+
+import UIKit
+
+class HelpViewController: UIViewController {
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/HelpViewController.swift:10]: same

File: homeWorkApp/homeWorkApp/Controllers/HelpViewController.swift [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+    
+    private var label3 = UILabel()
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/HelpViewController.swift:12]: same

File: homeWorkApp/homeWorkApp/Controllers/HelpViewController.swift [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+        super.viewDidLoad()
+        
+        style()
+        layout()
+    }
+    
+    private func style() {
+        view.backgroundColor = .white
+        title = "Help"
+        label3.textColor = .black
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/HelpViewController.swift:24]: same

File: homeWorkApp/homeWorkApp/Controllers/HomeViewController.swift [hunk 11]
```diff
@@ -0,0 +1,38 @@
+//
+//  HomeViewController.swift
+//  homeWorkApp
+//
+//  Created by Намик on 5/13/22.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/HomeViewController.swift:10]: same

File: homeWorkApp/homeWorkApp/Controllers/HomeViewController.swift [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+    
+    private var label1 = UILabel()
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/HomeViewController.swift:12]: same

File: homeWorkApp/homeWorkApp/Controllers/HomeViewController.swift [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+        super.viewDidLoad()
+        
+        style()
+        layout()
+    }
+    
+    private func style() {
+        view.backgroundColor = .white
+        title = "Home"
+        label1.textColor = .black
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/HomeViewController.swift:24]: same

File: homeWorkApp/homeWorkApp/Controllers/MainTabBarController.swift [hunk 14]
```diff
@@ -0,0 +1,51 @@
+//
+//  MainTabBarController.swift
+//  homeWorkApp
+//
+//  Created by Намик on 5/12/22.
+//
+
+import UIKit
+
+class MainTabBarController: UITabBarController {
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/MainTabBarController.swift:10]: same

File: homeWorkApp/homeWorkApp/Controllers/MainTabBarController.swift [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+    
+    private func setupTabBar() {
+        let homeViewController = createNavController(vc: HomeViewController(), itemName: "Home", itemImage: "house")
+        let contactsViewController = createNavController(vc: ContactsViewController(), itemName: "Contacts", itemImage: "person.circle")
+        let helpViewController = createNavController(vc: HelpViewController(), itemName: "Help", itemImage: "bell")
+        let aboutViewController = createNavController(vc: AboutViewController(), itemName: "About", itemImage: "star")
+        let settingsViewController = createNavController(vc: SettingsViewController(), itemName: "Settings", itemImage: "gear")
+        
+        viewControllers = [homeViewController, contactsViewController, helpViewController, aboutViewController, settingsViewController]
+
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/MainTabBarController.swift:41]: remove unnecessary empty space line

File: homeWorkApp/homeWorkApp/Controllers/SettingsViewController.swift [hunk 16]
```diff
@@ -1,23 +1,22 @@
...
 //  homeWorkApp
 //
-//  Created by Намик on 5/12/22.
+//  Created by Намик on 5/13/22.
 //
 
 import UIKit
 
-class ViewController: UIViewController {
+class SettingsViewController: UIViewController {
```
Comment (N-ihad) [inline homeWorkApp/homeWorkApp/Controllers/SettingsViewController.swift:10]: same

**Review by N-ihad — CHANGES_REQUESTED** [review]

File: homeWorkApp/homeWorkApp/Controllers/MainTabBarController.swift (hunk 14 above)
Comment (N-ihad) [reply homeWorkApp/homeWorkApp/Controllers/MainTabBarController.swift:10]: Final here as well

**Review by N-ihad — APPROVED** [review]

**Pull request merged**